Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/LocalData.kt

```diff
@@ -474,38 +474,6 @@ object LocalData {
 
     fun submissionWasSuccessful(): Boolean = numberOfSuccessfulSubmissions() >= 1
 
-    fun testGUID(): String? = getSharedPreferenceInstance().getString(
```

Comment:
Are these not needed anymore (testGUID, authCode, teletan)? I do not see an replacement

Reply:
Seems they were previously already unused. `testGUID` was still being called but only with `testGUID(null)`.

![Screenshot from 2020-12-07 19-37-07](https://user-images.githubusercontent.com/1439229/101390942-dea3ff80-38c3-11eb-8c97-bd96abc20036.png)

Reply:
yes, these have been removed by a prior refactoring and were only used by the tests and resetting them to null, i stumbled upon this while i was fixing unit tests and decided to remove them